src/patternfly/components/Badge/badge.scss

```
@@ -36,6 +39,16 @@
   background-color: var(--#{$badge}--BackgroundColor);
   border-radius: var(--#{$badge}--BorderRadius);
 
+  &::after {
+    position: absolute;
+    inset: 0;
+    pointer-events: none;
+    content: "";
+    border: var(--#{$badge}--BorderWidth) solid;
+    border-color: var(--#{$badge}--BorderColor);
```

Color can also go in the shorthand

```suggestion
    border: var(--#{$badge}--BorderWidth) solid var(--#{$badge}--BorderColor);
```